api-lib/circleAdmin.ts

```diff
@@ -37,9 +42,9 @@ const middleware =
       }
       throw new UnauthorizedError('User not circle admin', err);
     }
-    // the admin role is validated early by zod
 
-    await handler(req, res);
+    // Reject request by if not validated above
+    throw new UnauthorizedError('User not circle admin');
```

Comment:
In it's current state, all traffic from website will hit this error, since no hasura-level admin authentication is possible. This is why E2E tests are failing right now.